frontend/src/hooks/analytics/analytics.ts

```diff
@@ -0,0 +1,44 @@
+import Analytics from "analytics-node";
+import { uniqueId } from "lodash";
+import { getConfig } from "../../libraries/config";
+
+export interface Dict {
+  [key: string]: any;
+}
+
+let segment: Analytics;
+let anonymousId: string;
+
+function init() {
+  if (segment)
+    return console.log("Segment Analytics has already been initialized");
+  anonymousId = uniqueId(Date.now() + "_");
+  //flushAt=1 is useful for testing new events
+  const {
+    publicRuntimeConfig: { segmentWriteKey },
+  } = getConfig();
+  const options = process.env.NODE_ENV === "development" ? { flushAt: 1 } : {};
+  segment = new Analytics(segmentWriteKey, options);
+}
+
+function track(eventLabel: string, properties?: Dict) {
+  segment.track({
+    anonymousId,
+    event: eventLabel,
+    properties,
+  });
+}
+
+function identify() {
+  segment.identify({
+    userId: anonymousId,
+  });
+}
+
+const fns = {
```

Comment:
Why not just export `identify`, `init` and `track`?

Reply:
I was following the original pattern of this file from the console. Perhaps they required name exports?

Reply:
Yep, we're good with named exports in Explorer :)